dallinger/recruiters.py

```diff
@@ -872,6 +903,42 @@ def external_submission_url(self):
         self.prolificservice.log_response(response)
         return response
 
+    def screen_out_allowed(self, participants: list[Participant]):
+        """
+        Check which participants in a list of participants can be screened out.
+
+        Returns a list of participants that can be screened out.
+        """
+        study = self.prolificservice.get_study(self.current_study_id)
+
+        if not study.get("is_custom_screening", False):
+            raise ProlificRecruiterException(
+                f"Prolific study (ID {self.current_study_id}) doesn't allow screening-out of participants"
```

Comment:
Update the error message to tell the user they need to set `is_custom_screening` to `True` (in config.txt?)